packages/fiber/src/core/renderer.ts

```diff
@@ -408,7 +408,14 @@ function createRenderer<TCanvas>(roots: Map<TCanvas, Root>) {
       } else if (child.attachObject) {
         delete parentInstance[child.attachObject[0]][child.attachObject[1]]
       } else if (child.attach && !is.fun(child.attach)) {
-        parentInstance[child.attach] = null
+        // Prefer to not unset meshes' material/geometry.
+        if (parentInstance.isMesh && child.attach === 'material') {
+          parentInstance.material = new THREE.MeshBasicMaterial()
+        } else if (parentInstance.isMesh && child.attach === 'geometry') {
+          parentInstance.geometry = new THREE.BufferGeometry()
+        } else {
+          parentInstance[child.attach] = null
+        }
```

Comment:
Can we swap instances here or set visible to false?

Reply:
Actually, it looks like the suspended material isn't recreated nor attached to the scene once unsuspended. This may be a deeper issue.